src/test/java/com/github/cameltooling/lsp/internal/completion/CamelCompletionInsertAndReplaceTest.java

```diff
@@ -58,7 +58,7 @@ void testAttribute() throws Exception {
 		Position positionBeforeBufferSizeAttribute = new Position(0, 45);
 		CompletableFuture<Either<List<CompletionItem>, CompletionList>> completions = getCompletionFor(camelLanguageServer, positionBeforeBufferSizeAttribute);
 		List<CompletionItem> items = completions.get().getLeft();
-		assertThat(items).hasSize(13);
+		assertThat(items).hasSize(14);
```

Comment:
Why this change?

Reply:
the size is based on the number of attributes of a Camel component. We retrieve this information from the Camel Catalog. The new default Camel Catalog (3.18) contains one more attribute.

For some tests, we are creating specific Camel components to avoid being tide with a version fo the Camel catalog. For some others we are tied to the version of the Camel catalog.

Advantage of using provided catalog is that we are closer to reality end users are facing. The drawback is that we are affected in test when there are modifications in test like here.

Advantage of creatign specific component for the test si that we are not tied to a Camel version. The drawback is that we are not as close as real world for end users.

ot easy to find the right balance the two. So currently, we have a mix of the two to nto be hit too hard by the drawbacks of each of them.